fog/ingest/enclave/src/lib.rs

```diff
@@ -101,9 +119,9 @@ impl IngestSgxEnclave {
 
         sgx_enclave
             .enclave_init(params)
-            .expect("enclave_init failed");
+            .map_err(NewEnclaveError::Init)?;
```

Comment:
For this kind of thing, if you implement `From<EnclaveError> for NewEnclaveError`, you can do `sgx_enclave.enclave_init(params)?;` and it will do the right thing. This is generally the preferred method for this unless there's some hairy thing where it's not always a 1:1 mapping between the error that you're getting and the one you want to return.